Source/Core/InputCommon/GCAdapter.cpp

```diff
@@ -115,11 +114,23 @@ static int HotplugCallback(libusb_context* ctx, libusb_device* dev, libusb_hotpl
       std::lock_guard<std::mutex> lk(s_init_mutex);
       AddGCAdapter(dev);
     }
+    else if ((s_status < 0) && (s_detect_callback != nullptr))
```

Comment:
Those additional parentheses aren't really necessary, you can remove those (this also applies to pretty much all the other places)